lib/jsonapi/utils.rb

```diff
@@ -6,9 +6,12 @@ module JSONAPI
   module Utils
     def self.included(base)
       if base.respond_to?(:helper_method)
-        base.before_action :setup_request, :check_request
         base.helper_method :jsonapi_render, :jsonapi_serialize
       end
+
+      if base.respond_to?(:before_action)
```

Comment:
@tiagopog look reasonable? 

We don't need `jsonapi_render` and `_serialize` as helper methods (i.e. to be used in views if necessary) since we're API only, but we *do* need those `before_action`s to fire. So I suggest separating them per this PR.